### PR TITLE
Removing old refresh_stats and renaming refresh_stats_rq

### PR DESCRIPTION
**Pull request by phlax**
After some discussion we thought it would be ok to drop calculate_wordcount for now (touch #3548)

This commit simply renames `refresh_stats_rq` to `refresh_stats`

I will update docs...

### CODE REVIEW

**Comment by unho** [comment]
lgtm

**Comment by phlax** [comment]
@unho docs updated

**Comment by unho** [comment]
Please include this change in the release notes and squash both commits.

File: docs/server/commands.rst [hunk 1]
```diff
@@ -88,49 +88,6 @@ refresh_stats
...
-.. django-admin:: refresh_stats_rq
-
-refresh_stats_rq
-^^^^^^^^^^^^^^^^
-
-.. versionadded:: 2.7
-
-Refreshes all calculated statistics ensuring that they are up-to-date.
-
 A background process will create a task for every file to make sure calculated
```
Comment (unho) [inline docs/server/commands.rst]: A few lines below there is a note https://github.com/translate/pootle/pull/4049/files#diff-32d9c83230475a8ff37b1695b1606581R97 referring to the old `refresh_stats`, a note that I don't think it is necessary anymore.

**Comment by phlax** [comment]
@unho updated

File: docs/server/commands.rst (hunk 1 above)
Comment (unho) [inline docs/server/commands.rst]: Done.

**Comment by unho** [comment]
lgtm

**Pull request merged**